pymeasure/instruments/lakeshore/lakeshore_base.py

```diff
@@ -35,31 +36,41 @@
 
 class LakeShoreTemperatureChannel(Channel):
     """ Temperature input channel on a lakeshore temperature monitor. Reads the temperature in
-    kelvin, celcius, or sensor units. Also provides a method to block the program until a given
+    kelvin, celsius, or sensor units. Also provides a method to block the program until a given
     stable temperature is reached.
     """
 
     kelvin = Instrument.measurement(
         'KRDG? {ch}',
         """Read the temperature in kelvin from a channel."""
     )
-    celcius = Instrument.measurement(
+    celsius = Instrument.measurement(
         'CRDG? {ch}',
-        """Read the temperature in celcius from a channel."""
+        """Read the temperature in celsius from a channel."""
     )
     sensor = Instrument.measurement(
         'SRDG? {ch}',
         """Read the temperature in sensor units from a channel."""
     )
 
+    @property
+    def celcius(self):
```

Comment:
Is this alright @BenediktBurger?

What version should I put under `.. deprecated:: {next_version}`?

Reply:
The current version is 0.13.1. The next release will be 0.14.0

Reply:
You might add single ticks around the 'celcius" and' celsius' in the deprecation warning (in "warn").

Good idea to use a plain property.

Reply:
Done.